.github/workflows/cd.yml

```diff
@@ -100,3 +149,45 @@ jobs:
           "zebrad-${{ env.GITHUB_HEAD_REF_SLUG || env.GITHUB_REF_SLUG }}" \
           --version template="zebrad-${{ env.GITHUB_HEAD_REF_SLUG || env.GITHUB_REF_SLUG }}-${{ env.GITHUB_SHA_SHORT }}" \
           --region "${{ env.REGION }}"
+
+  deploy-instance:
+    name: Deploy single instance
+    needs: build
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    if: github.event_name == 'workflow_dispatch'
+
+    steps:
+      - uses: actions/checkout@v2.4.0
+        with:
+          persist-credentials: false
+
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v4
+
+      # Setup gcloud CLI
+      - name: Authenticate to Google Cloud
+        id: auth
+        uses: google-github-actions/auth@v0.5.0
+        with:
+          credentials_json: ${{ secrets.GOOGLE_CREDENTIALS }}
+
+      - name: Set up gcloud SDK environment
+        uses: google-github-actions/setup-gcloud@v0.4.0
+        with:
+          project_id: ${{ env.PROJECT_ID }}
+
+      # Create instance template from container image
+      - name: Manual deploy of a single instance running zebrad
+        run: |
+          gcloud compute instances create-with-container "zebrad-${{ env.GITHUB_HEAD_REF_SLUG || env.GITHUB_REF_SLUG }}-${{ env.GITHUB_SHA_SHORT }}" \
+          --boot-disk-size 100GB \
+          --boot-disk-type=pd-ssd \
+          --container-stdin \
+          --container-tty \
+          --container-image ${{ env.GAR_BASE }}/${{ env.GITHUB_HEAD_REF_SLUG || env.GITHUB_REF_SLUG }} \
```

Comment:
Same here, should we include the `SHORT_SHA`?

Reply:
Fixed: https://github.com/ZcashFoundation/zebra/pull/3431/commits/e88f443ec0f4f965c1178609916e8fd2dc503e0a